dbt/adapters/impala/connections.py

```diff
@@ -129,7 +131,8 @@ def open(cls, connection):
                     user=credentials.username,
                     password=credentials.password,
                     use_ssl=credentials.use_ssl,
-                    http_path=credentials.http_path
+                    http_path=credentials.http_path,
+                    retries=credentials.retries
```

Comment:
If we don't add explicit code for retires here will the underlying [dbapi](https://github.com/cloudera/impyla/blob/master/impala/dbapi.py#L47) not automatically retry 3 times .

Reply:
Yes, it will. But this reads user specified retries value from profiles.yml and passes it explicitly to this method.

Reply:
Gotcha.